mail_partner_notification/models/notification.py

```diff
@@ -31,6 +31,8 @@ def _notify(self, message_id, partners_to_notify=None,
                 force_send=False, user_signature=True):
         if 'notify' in self.env.context and not self.env.context['notify']:
             partners_to_notify = self.env.context.get('partners_to_notify', [])
+        elif 'notify' in self.env.context and self.env.context['notify']:
```

Comment:
I think this line is not needed since it does the same as 
```python
if 'notify' in self.env.context and not self.env.context['notify']:
    partners_to_notify = self.env.context.get('partners_to_notify', [])
```
if **notify** isn't true then replace partners else if **notify** is true replace partners. Partners should be replaced only when 
```python
notify == False
```